application-task-manager-ui/src/main/resources/TaskManager/Code/TaskReportResultPage.xml

```diff
@@ -37,123 +37,42 @@
+  #set ($list = [])
+  #foreach ($tag in $request.getParameterValues('tasktag'))
+    #set ($discard = $list.add($tag))
   #end
-  #set($discard = $map.put('offset', $mathtool.add($offset, 1)))
-##
-##
-  #set($rows = [])
-  #set($lastDoc = '')
-  #set($lastObjList = '')
-  #foreach($item in $items)
-    #set($rowDocRef = $item.documentReference)
-    #set($objId = $item.id)
-    #set($rowDoc = $xwiki.getDocument($rowDocRef))
-    #set($row = {})
-    #if ($rowDocRef)
-      #set ($row = {
-        'doc_title': $rowDoc.translatedDocument.plainTitle,
-        'doc_viewable': true,
-        'doc_fullName': $services.model.serialize($rowDocRef, 'local'),
-        'doc_space': $services.model.serialize($rowDocRef.parent, 'local'),
-        'doc_url': $xwiki.getURL($rowDocRef),
-        'doc_space_url': $xwiki.getURL($rowDocRef.parent),
-        'doc_wiki': $rowDocRef.wikiReference.name,
-        'doc_wiki_url': $xwiki.getURL($rowDocRef.wikiReference),
-        'doc_hasadmin': $xwiki.hasAdminRights(),
-        'doc_hasedit': $services.security.authorization.hasAccess('edit', $rowDocRef),
-        'doc_hasdelete': $services.security.authorization.hasAccess('delete', $rowDocRef),
-        'doc_edit_url': $xwiki.getURL($rowDocRef, 'edit'),
-        'doc_copy_url': $xwiki.getURL($rowDocRef, 'view', 'xpage=copy'),
-        'doc_delete_url': $xwiki.getURL($rowDocRef, 'delete'),
-        'doc_rename_url': $xwiki.getURL($rowDocRef, 'view', 'xpage=rename&amp;step=1')
-      })
-      #set($discard = $row.put('description', $rowDoc.getRenderedContentRestricted($item.description, $rowDoc.syntax)))
-      #set($discard = $row.put('createDate', $datetool.format($services.taskmanager.configuration.displayDateFormat, $item.createDate)))
-      #set($discard = $row.put('deadline', $datetool.format($services.taskmanager.configuration.displayDateFormat, $item.deadline)))
-      #set($discard = $row.put('completeDate', $datetool.format($services.taskmanager.configuration.displayDateFormat, $item.completeDate)))
-      #set($discard = $row.put('completed', $services.localization.render("taskmanager.livetable.completed.${item.completed}")))
-      #addUserDisplay($row 'creator' $item.creator)
-      #addUserDisplay($row 'assignees' $item.assignees)
-    #else
-      #set($discard = $row.put('description', $services.localization.render('taskmanager.livetable.noRights')))
+  #set ($discard = $params.put("taskTagList", $list))
+#end
+
+#if ("$!request.space" != '')
+  #foreach($space in $request.getParameterValues('space'))
+    #if ($foreach.count > 1)
+      #set ($spaceWhere = "${spaceWhere} OR ")
     #end
-    #set($discard = $rows.add($row))
+    #set ($paramVal = "$!space.replaceAll('([%_!])', '!$1')%")
+    #set ($spaceWhere = "${spaceWhere}doc.fullName like :docspace${foreach.count} escape '!'")
+    #set ($discard = $params.put("docspace${foreach.count}", $paramVal))
```

Comment:
This should be explained because it's not obvious at all why you don't use the dedicated API for like operator. BTW, did you report the JIRA issue for platform (regarding the ``in`` operator)? Add the link to the JIRA issue afterwards.

Reply:
The issue did not reproduce in versions greater than 13.0. It might have been solved by this jira ticket. https://jira.xwiki.org/browse/XWIKI-18193 but I am not entirely sure.